zerver/management/commands/generate_realm_creation_link.py

```diff
@@ -16,6 +19,15 @@ class Command(BaseCommand):
 
     def handle(self, *args, **options):
         # type: (*Any, **Any) -> None
+
+        try:
+            # first check if the db has been initalized
+            Realm.objects.first()
+
+        except ProgrammingError:
+            print("The Zulip database does not appear to exist. Have you run initialize-database?")
+            sys.exit(1)
```

Comment:
This has more blank lines than it should, otherwise looks great!  I'll just remove the lines and merge.